add beta page (#64)
amplication/docs: docs/getting-started/beta-version.md

---
id: beta-version
title: Amplication Beta Version
sidebar_label: Beta Version
slug: /beta-version
---

# Amplication Beta Version

Amplication is currently in Beta. It means that we are still working on essential features like production-ready hosting, migrations, and stability to our console.

> <br />
> <br />

Reviewer: ```suggestion

```

> How does it effect you? Well...., it mostly doesn't.<br />

Reviewer: ```suggestion
How does it effect you? Well...., it mostly doesn't.

```

Every app generated using Amplication platform contains popular, documented, secured, and supported production-ready open source components & packages ([read more about our stack here](https://docs.amplication.com/docs/getting-started#technologies)). Your app is stable, scalable, and production-ready you can deploy and rely on.

> <br /><br />

Reviewer: ```suggestion

```

> Please note, Our free sandbox environment is limited in resources, not backed up, and can reset at any time therefore we do not recommend using the free sandbox environment as a production environment.

Reviewer: ```suggestion
## Amplication Sandbox Environment
Please note, Our free sandbox environment is limited in resources, not backed up, and can reset at any time therefore we do not recommend using the free sandbox environment as a production environment.
```

How does it effect you? Well...., it mostly doesn't.

> Every app generated using Amplication platform is built on top of packages that are all production-ready, open-source, and well adopted by many projects around the world. These packages are independently documented, supported, and maintained by their parent company on regular basis. ([read more about our stack here](https://docs.amplication.com/docs/getting-started#technologies)). Your app is stable, scalable, and production-ready you can deploy and rely on.

Reviewer: ```suggestion
Every app generated using Amplication platform is built on top of well-adopted, open-source packages, that are all production-ready. These packages are independently documented, supported, and maintained by their own teams on regular basis. ([read more about our stack here](https://docs.amplication.com/docs/getting-started#technologies)). Your app is stable, scalable, and production-ready you can deploy and rely on.
```
